webui/react/src/pages/Settings/UserManagement.tsx

```diff
@@ -163,7 +163,7 @@ const UserManagement: React.FC = () => {
   const fetchUsers = useCallback((): void => {
     if (!settings) return;
 
-    usersStore.ensureUsersFetched(canceler, apiConfig);
+    usersStore.ensureUsersFetched(canceler, apiConfig, true);
```

Comment:
Isn't the issue here that we're not updating the store when adding a user?

Reply:
That's part of it, but just adding the user to the store wouldn't update the table. Currently the store uses the (stringified) fetch config as a key to return an array of user ids, which are then matched up to users in the store. I could add the ability to modify that id array, but that seems like a footgun waiting to happen. I'm open to suggestions for better ways to approach this though.